Validation/HGCalValidation/plugins/HGCalRecHitValidation.cc

```diff
@@ -88,6 +96,17 @@ class HGCalRecHitValidation : public DQMEDAnalyzer {
 
 HGCalRecHitValidation::HGCalRecHitValidation(const edm::ParameterSet& iConfig)
     : nameDetector_(iConfig.getParameter<std::string>("DetectorName")),
+      caloGeomToken_(esConsumes<CaloGeometry, CaloGeometryRecord>()),
+      ee_geometry_token_(esConsumes(edm::ESInputTag("", "HGCalEESensitive"))),
+      hesil_geometry_token_(esConsumes(edm::ESInputTag("", "HGCalHESiliconSensitive"))),
+      hesci_geometry_token_(esConsumes(edm::ESInputTag("", "HGCalHFNoseSensitive"))),
```

Comment:
If I read the code correcly, this could be simplified to
```suggestion
      geometry_token_(esConsumes(edm::ESInputTag("", nameDetector_))),
```
after which the new code would be correspond more closely to the old code. (same for the beginRun token)